src/diffcalc_API/models/UBCalculation.py

```diff
@@ -41,3 +41,7 @@ class editOrientationParams(BaseModel):
     xyz: Optional[Tuple[float, float, float]] = None
     position: Optional[Tuple[float, float, float, float, float, float]] = None
     tagOrIdx: Union[int, str]
+
+
+class deleteParams(BaseModel):
```

Comment:
Is there a reason this is `lowerCamelCase`?

Reply:
I'm not 100% sure on the convention for camel casing here. I think all my variables are `lowerCamelCase`, my functions `have_underscores` and that's all I've done really. If typically, classes are written with `UpperCamelCase` I'll change that.

Reply:
That's the convention, definitely worth reading through [PEP8](https://peps.python.org/pep-0008/) if you're unfamiliar